grammar.js

```diff
@@ -117,32 +84,109 @@ module.exports = grammar({
         local: () => "local",
         tailstrict: () => "tailstrict",
 
+        // Types
+        number: ($) => $._number,
+        string: ($) => $._string,
```

Comment:
I've linked $.number and $.string here so they show up in the right order but all the details can stay at the bottom to reduce the diff.